Wrapping/Python/Packaging/setup.py.in

```diff
@@ -43,28 +46,43 @@ def get_pep386version():
 
     return version
 
+class build_ext(_build_ext):
+    """ Override standard command class to build an extension, to
+    simply copy an existing compiled library into the packaging
+    directory structure.
+    """
 
-# HACK
-# We are using the "data_files" parameter to package up the SimpleITK
-# libraries. However distribute does not detect this as an extension
-# and does not build a egg with the platform name in the egg file
-# name. So we forcable overload a method so that distutils, always
-# reports that we are always an extension.
-def always_has_ext_modules (self):
-    """Always returns true, to force extensimon behavior in packaging."""
-    return True
-Distribution.has_ext_modules =  always_has_ext_modules
+    def build_extension(self, ext):
+        """
+        """
+        from distutils.errors import DistutilsSetupError
 
+        sources = ext.sources
+        if sources is None or not len(sources) == 1:
+            raise DistutilsSetupError( "Expected only one compiled library." )
+
+        expected_ext_filename = os.path.split(self.get_ext_filename(ext.name))[1]
+
+        if not expected_ext_filename == os.path.split(sources[0])[1]:
+            raise  DistutilsSetupError(
+                "Expected compiled extension named {0}, not {1}"
+                .format( expected_ext_filename, sources[0] ) )
+
+        ext_dir = os.path.dirname( self.get_ext_fullpath(ext.name) )
+
+        abs_sources =  map(os.path.abspath, sources)
+
+        self.copy_file(abs_sources[0], ext_dir)
 
 
 setup(
     name = 'SimpleITK',
     version = get_pep386version(),
     author = 'Insight Software Consortium',
     author_email = 'insight-users@itk.org',
+    ext_modules=[Extension('SimpleITK._SimpleITK', [r'@SimpleITK_RELATIVE_BINARY_MODULE@'])],
     packages= ['SimpleITK'],
     package_dir = {'SimpleITK':r'@SimpleITK_PYTHON_PACKAGE_DIR@'},
-    package_data = {'SimpleITK':[r'@SimpleITK_RELATIVE_BINARY_MODULE@']},
     download_url = r'https://www.itk.org/SimpleITKDoxygen/html/PyDownloadPage.html',
     platforms = [],
     description = r'Simplified interface to the Insight Toolkit for image registration and segmentation',
```

Comment:
possibly change string to r'Simplified interface to the Insight Segmentation and Registration Toolkit (ITK)'